README.md

```diff
@@ -6,66 +10,70 @@ Use Twilio to send SMS alerts so that you never miss a critical issue.
 
 [Read the full tutorial here](https://www.twilio.com/docs/tutorials/walkthrough/server-notifications/node/express)!
 
-## Running the Project on Your Machine
+## Local development
 
-To run this project on your computer, download or clone the source. You will 
-also need to download and install either [Node.js](http://nodejs.org/) 
-or [io.js](https://iojs.org/en/index.html), both of which should also install 
-[npm](https://www.npmjs.com/). 
+To run this project on your computer you will
+ need to download and install either [Node.js](http://nodejs.org/)
+or [io.js](https://iojs.org/en/index.html), both of which should also install
+[npm](https://www.npmjs.com/).
 
-You will also need to [sign up for a Twilio account](https://www.twilio.com/try-twilio) 
+You will also need to [sign up for a Twilio account](https://www.twilio.com/try-twilio)
 if you don't have one already.
 
-### Install Dependencies
+1. First clone this repository and `cd` into it.
 
-Navigate to the project directory in your terminal and run:
+   ```bash
+   $ git clone git@github.com:TwilioDevEd/server-notifications-node.git
+   $ cd server-notifications-node
+   ```
 
-```bash
-npm install
-```
+1. Next, open `.env.example` at the root of the project and update it with
+   values from your
+   [Twilio account](https://www.twilio.com/user/account/voice-messaging)
+   and local configuration. Save the file as `.env`.  You'll need to set
+   `TWILIO_AUTH_TOKEN`, `TWILIO_ACCOUNT_SID`, and `TWILIO_NUMBER`.
 
-This should install all of our project dependencies from npm into a local 
-`node_modules` folder.
+   For the `TWILIO_NUMBER` variable you'll need to provision a new number
+   in the
+   [Manage Numbers page](https://www.twilio.com/user/account/phone-numbers/incoming)
+   under your account. The phone number should be in
+   [E.164 format](https://www.twilio.com/help/faq/phone-numbers/how-do-i-format-phone-numbers-to-work-internationally)
 
-### Configuration
+    Run `source .env` to export the environment variables.
 
-Next, open `.env.example` at the root of the project and update it with
-values from your
-[Twilio account](https://www.twilio.com/user/account/voice-messaging)
-and local configuration. Save the file as `.env`.  You'll need to set
-`TWILIO_AUTH_TOKEN`, `TWILIO_ACCOUNT_SID`, and `TWILIO_NUMBER`.
+1. Navigate to the project directory in your terminal and run:
 
-For the `TWILIO_NUMBER` variable you'll need to provision a new number
-in the
-[Manage Numbers page](https://www.twilio.com/user/account/phone-numbers/incoming)
-under your account. The phone number should be in
-[E.164 format](https://www.twilio.com/help/faq/phone-numbers/how-do-i-format-phone-numbers-to-work-internationally)
+    ```bash
+    npm install
```

Comment:
One thing that makes easy to read something is some homogeneous style. So all the bash instructions should be prepended by a `$`.

Additionally, would you mind updating this template please? https://github.com/twilio/docs-guidelines/blob/master/templates/README/node/express/README.md -- ping me if you have any questions.